docs/src/main/sphinx/functions/datasketches.md

```diff
@@ -0,0 +1,54 @@
+# DataSketches functions
+
+DataSketches is a high-performance library of stochastic streaming
+algorithms, commonly called sketches. Sketches are small, stateful programs
+that process massive data as a stream and can provide approximate answers,
+with mathematical guarantees, much faster than traditional exact methods.
+The DataSketches functions allow querying the fast and memory-efficient
+[Apache DataSketches](https://datasketches.apache.org/docs/Community/Research.html)
+from Trino. Support for the
+[Theta Sketch framework](https://datasketches.apache.org/docs/Theta/ThetaSketchFramework.html)
+is available through {func}`theta_sketch_union` and
+{func}`theta_sketch_estimate`, typically used in `COUNT DISTINCT` queries.
+DataSketches can be created with Hive or Pig using their respective sketch
+APIs.
+
+To use these functions, configure a catalog with the
+``datasketches`` connector (for example, name the catalog ``datasketches``)
+and either qualify function calls with ``datasketches.theta.`` or add
+``datasketches.theta`` to the session ``path``.
+
+Note: Trino does not create new sketches. Build the Theta sketches in your
+ingest pipeline or another engine (e.g., Spark or Hive) and store the
+serialized sketch bytes as a ``VARBINARY`` column. Trino functions operate on those serialized sketches.
```

Comment:
DataSketches supports many sketch families and serialization format is to some extend configurable. This should say what's supported.

Reply:
Here we only support Theta sketches